scripts/ecs/appsignals/setup-ecs-demo.sh

```diff
@@ -3,679 +3,57 @@ set -ex
-          aws elbv2 delete-listener \
-              --listener-arn $listener_arn > /dev/null
-          echo "Listener deleted."
-      fi
-
-      # Delete the target group
-      aws elbv2 delete-target-group \
-          --target-group-arn $api_gateway_target_group_arn > /dev/null
-      echo "Target group deleted."
-
-      echo "All resources deleted."
+function run_cdk() {
+  echo "Running CDK..."
+  # jump to the cdk folder, run the cdk commands, and then jump back to current folder
+  pushd ../../../cdk/ecs >/dev/null
+  ./ecs-cdk.sh $1
+  popd >/dev/null
 }
 
 function delete_resources() {
-    echo "Deleting resources..."
-
-    while true; do
-      remaining_services=$(aws ecs list-services --cluster ${CLUSTER} --query 'serviceArns' --output text)
-      if [ -z "$remaining_services" ]; then
-        echo "All services deleted."
-        break
-      else
-        echo "Still waiting for services to be deleted..."
-        sleep 5
-      fi
-    done
-
-    aws ecs delete-cluster --cluster ${CLUSTER}
-
-    namespace_id=$(aws servicediscovery list-namespaces \
-             --query "Namespaces[?Name=='ecs-petclinic'].Id" --output text)
-    aws servicediscovery delete-namespace --id $namespace_id
+  echo "Deleting resources..."
 
-    # Detach and delete IAM policies for ECS Task role
-    task_role_policy_arns=("arn:aws:iam::aws:policy/AmazonKinesisFullAccess" "arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess" "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy" "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole" "arn:aws:iam::aws:policy/AmazonECS_FullAccess" "arn:aws:iam::aws:policy/AmazonSQSFullAccess" "arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess" "arn:aws:iam::aws:policy/AmazonRDSFullAccess" "arn:aws:iam::aws:policy/AmazonS3FullAccess" "arn:aws:iam::aws:policy/AmazonBedrockFullAccess")
-    for arn in "${task_role_policy_arns[@]}"
-    do
-      echo $arn
-      aws iam detach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn $arn
-    done
-    aws iam delete-role --role-name $IAM_TASK_ROLE_NAME
+  run_cdk destroy
 
-    # Detach and delete IAM policies for ECS Task Execution role
-    task_execution_role_policy_arns=("arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy" "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy")
-    for arn in "${task_execution_role_policy_arns[@]}"
-    do
-      echo $arn
-      aws iam detach-role-policy --role-name $IAM_EXECUTION_ROLE_NAME --policy-arn $arn
-    done
-    aws iam delete-role --role-name $IAM_EXECUTION_ROLE_NAME
+  # delete resources created by the sample app itself
+  aws sqs delete-queue --queue-url $(aws sqs get-queue-url --queue-name apm_test --query 'QueueUrl' --output text)
+  aws kinesis delete-stream --stream-name apm_test
+  aws dynamodb delete-table --table-name apm_test
+  # TODO: Reimplement resources deletion as current Sample App in ECS has issues creating these resources in the region.
```

Comment:
Is this comment still valid?

Reply:
Yes, BillingInfo and PetClinicPayment are not created as region isn't specified. It is still an issue with ECS Sample App.

Reply:
Thanks. I have added `REGION` variable to billing-service container to create the BillingInfo table in the correct region.